Sources/SwiftDocC/Model/Rendering/RenderNode/RenderMetadata.swift

```diff
@@ -163,6 +172,21 @@ extension RenderMetadata: Codable {
         /// but have no authoring support at the moment.
         public let relatedModules: [String]?
     }
+    
+    /// Describes the location of the topic's source code, hosted by a source service.
+    public struct Source: Codable, Equatable {
```

Comment:
Maybe this should be `Location` or `Declaration`? We use `Location` in SymbolKit it looks like: https://github.com/apple/swift-docc-symbolkit/blob/main/openapi.yaml#L290.

I think `Source` is a good term for describing the entire collection of a project's source code but I'm not sure it's right for describing a specific symbol's declaration.

Reply:
I'm leaning towards being ok with "source" either referring to the source repository of the code in the 'semantic analysis' part of DocC and it referring to a specific symbol's remote source file when rendering each topic individually. I've renamed this to `remoteSource` to avoid confusion with the source of the symbol in a local file.

Reply:
(fyi @dobromir-hristov, if we go with this name change, we'll need to update the DocC Render PR to update the property name in the metadata object)

Reply:
No problem, just remind me in the PR with the final name :)